gateway/src/inference/providers/openai.rs

```diff
@@ -1122,6 +1278,56 @@ impl<'a> TryFrom<OpenAIEmbeddingResponseWithMetadata<'a>> for EmbeddingProviderR
     }
 }
 
+#[derive(Debug, Deserialize)]
+struct OpenAIFileResponse {
+    id: String,
+}
+
+#[derive(Debug, Deserialize)]
+struct OpenAIBatchResponse {
+    id: String,
+    // object: String,
+    // endpoint: String,
+    // errors: OpenAIBatchErrors,
+    // input_file_id: String,
+    // completion_window: String,
+    // status: String,
+    // output_file_id: String,
+    // error_file_id: String,
+    // created_at: i64,
+    // in_progress_at: Option<i64>,
+    // expires_at: i64,
+    // finalizing_at: Option<i64>,
+    // completed_at: Option<i64>,
+    // failed_at: Option<i64>,
+    // expired_at: Option<i64>,
+    // cancelling_at: Option<i64>,
+    // cancelled_at: Option<i64>,
+    // request_counts: OpenAIBatchRequestCounts,
+    // metadata: HashMap<String, String>,
+}
+
+#[derive(Debug, Deserialize)]
+struct OpenAIBatchErrors {
+    // object: String,
+    // data: Vec<OpenAIBatchError>,
+}
+
+#[derive(Debug, Deserialize)]
+struct OpenAIBatchError {
+    // code: String,
+    // message: String,
+    // param: Option<String>,
+    // line: Option<i32>,
+}
```

Comment:
Is stuff like this used anywhere?

Reply:
this was for my reference for the next PR -- not used yet